tests/unit/test_index.py

```diff
@@ -308,7 +308,10 @@ def test_request_retries(caplog):
         # URL with something that looks like a drive letter, but is
         # not. The `:` should be quoted.
         ("https://localhost.localdomain/T:/path/",
-         "https://localhost.localdomain/T%3A/path/")
+         "https://localhost.localdomain/T%3A/path/"),
+        # URL with link to git.
+        ("git+ssh://example.com/path.git@1.0#egg=my-package-1.0.201810251140",
```

Comment:
I would include at least one character that needs to be quoted (e.g. "path space.git") so the example can show something being quoted in the same URL that `@` is not quoted. Also, you can simplify slightly by removing "10251140" from the end.

Reply:
Done